### DISCUSSION TITLE
Messstation Kornelimünster (2824300000100) Home Assistant NRW Hochwasser

### DISCUSSION THREAD

**Discussion by Bijouri** [stephan192/hochwasserportal#10] [General]
Hallo Stephan192,

ich wohne im Einzuggebiet der der Messstation Kornelimünster (2824300000100) 
https://www.elwasweb.nrw.de/elwas-web/data/ow/menge/pegel/pegelDetailsTab.xhtml?cid=4

Da ich vom NRW Hochwasser betroffen war, wolte ich ich mit Home Assistant Messstation Kornelimünster (2824300000100) 
die Messstation überwachen.
Bin auf deine Home Assistant Integration gestossen und habe beie Probleme die Daten der benachbarten Messstation
Mulartshütte NW_2824450000100 und Eschweiler NW_2824590000400 auszulesen, die mich nur indirekt betreffen.
die in deiner liste geführt werden.
Ich habe dann versucht mit dem eintrag NW_2824300000100 die Messstation Kornelimünster auszulesen ohne erfolg.

Ich vermute das das in der Integration die Messstation Kornelimünster eingeflegt werden mus, kanst du mir hier helfen oder gibt einen andren weg?

Beste Grüße Andreas

mein eintag configuration.yaml

# Mulartshütte (2824450000100)
  - platform: hochwasserportal
    pegel: NW_2824450000100
    level: True
    stage: True
    flow: True

# Eschweiler (2824590000400)    
  - platform: hochwasserportal
    pegel: NW_2824590000400
   level: True
   stage: True
   flow: True

# Kornelimünster (2824300000100)
  - platform: hochwasserportal
    pegel: NW_2824300000100
    level: True
    stage: True
    flow: True

**Comment by stephan192** [comment]
Meine Integration kennt nur Pegel die auf https://www.hochwasserzentralen.de/ gelistet sind.
Mit einem Skript aktualisiere ich von Zeit zu Zeit die Liste der Pegel unter https://github.com/stephan192/hochwasserportal/blob/main/pegel.md

Kornelimünster ist aktuell nicht gelistet, da kann ich nichts machen. Sobald der Pegel im LHP gelistet wird kann Du in mit meiner Integration abfragen. Warum er nicht gelistet ist musst du beim LHP fragen.

**Comment by stephan192** [comment]
Hallo @Bijouri da die Daten für NRW inzwischen direkt von https://hochwasserportal.nrw/lanuv/ abgeholt werden wird ab jetzt auf der Pegel NW_2824300000100 (KornelimuensterW / Inde) unterstützt.

**Comment by Bijouri** [comment]
Danke dir fürs bescheid geben, funktioniert einbahnfrei.
